cleverhans/future/tf2/utils_tf.py

```diff
@@ -0,0 +1,722 @@
+"""Utility functions for writing TensorFlow code"""
```

Comment:
I didn't realize this enormous file was added. From the look of it, it seems that you copied an existing file into future. I would say keep utils_tf.py to its bare minimum (we thought several times of not having a utils file since ends collecting all kind of unrelated stuff), like remove all the coded that's not needed by your current functions.